KarhooUISDK/UIComponents/AddressBarMVP/KarhooAddressBarView.swift

```diff
@@ -42,19 +42,23 @@ public class KarhooAddressBarView: UIView, AddressBarView {
     private let horizontalPadding: CGFloat
     private let hidePickUpDestinationConnector: Bool
     private let destinationIconSize: CGFloat = 18.0
+    private let keepPrebookHidden: Bool
 
     init(cornerRadious: CGFloat = 5.0,
          borderLine: Bool = false,
          dropShadow: Bool = true,
          verticalPadding: CGFloat = 0.0,
          horizontalPadding: CGFloat = 0.0,
-         hidePickUpDestinationConnector: Bool = false) {
+         hidePickUpDestinationConnector: Bool = false,
+         hidePrebookButton: Bool = false
+    ) {
         self.dropShadow = dropShadow
         self.borderLine = borderLine
         self.cornerRadious = cornerRadious
         self.verticalPadding = verticalPadding
         self.horizontalPadding = horizontalPadding
         self.hidePickUpDestinationConnector = hidePickUpDestinationConnector
+        self.keepPrebookHidden = hidePrebookButton
```

Comment:
What do you think about keeping the same name? I found it a bit confusing at first because one started with keep and the other with hide, and needed to read twice to make sure I understood the scope correctly.

Reply:
Maybe consider using `isPrebookHidden` to keep the naming aligned with iOS conventions.

Reply:
i will set both names to `hidePrebookButton`. `isPrebookHidden` is potentially unclear, since it can be understood as get-state instead of force-state property.